core/io/shared/tty.rb

```diff
@@ -1,25 +1,27 @@
 require File.expand_path('../../fixtures/classes', __FILE__)
 
 describe :io_tty, shared: true do
-  with_tty do
-    begin
-      # check to enabled tty
-      File.open('/dev/tty') {}
+  platform_is_not :windows do
+    with_tty do
+      begin
+        # check to enabled tty
+        File.open('/dev/tty') {}
 
-      # Yeah, this will probably break.
-      it "returns true if this stream is a terminal device (TTY)" do
-        File.open('/dev/tty') { |f| f.send(@method) }.should == true
+        # Yeah, this will probably break.
+        it "returns true if this stream is a terminal device (TTY)" do
+          File.open('/dev/tty') { |f| f.send(@method) }.should == true
+        end
+      rescue Errno::ENXIO
+        # workaround for not configured environment
       end
-    rescue Errno::ENXIO
-      # workaround for not configured environment
     end
-  end
 
-  it "returns false if this stream is not a terminal device (TTY)" do
-    File.open(__FILE__) { |f| f.send(@method) }.should == false
-  end
+    it "returns false if this stream is not a terminal device (TTY)" do
+      File.open(__FILE__) { |f| f.send(@method) }.should == false
+    end
```

Comment:
Yep, good catch @eregon, sorry I missed that - just pushed another commit to fix it.